src/Chat.tsx

```diff
@@ -26,6 +26,7 @@ export interface ChatProps {
     directLine?: DirectLineOptions,
     speechOptions?: SpeechOptions,
     locale?: string,
+    pastHistory?: Activity[],
```

Comment:
"past" and "history" mean the same thing - how about just `history`?

Reply:
I agree. I wanted to disambiguate from the history component.